README.md

```diff
@@ -47,12 +47,14 @@ target 'YourAwesomeProject' do
 
   permissions_path = '../node_modules/react-native-permissions/ios'
 
+  pod 'RNPermissions', :path => '../node_modules/react-native-permissions', :modular_headers => true
```

Comment:
this one is, I think, auto-linked and not necessary? Does something about this PR make it necessary now?

Reply:
This was needed to get around an error on pod install caused by mixing Swift and Objective C - there may be another way around it

Reply:
Removed now I've rewritten the handler in objective c d2d598fc5aa0bcfaf0aa2c186db65207f9061a01